src/components/SectionHeader/SectionHeader.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './SectionHeader.css';
+
+const SectionHeader = ({ header }) => (
+	<div className='SectionHeader'>
+		<h2>{header}</h2>
```

Comment:
Can this be `{children}` instead of `{header}`?

```javascript
// now
<SectionHeader header='Content here' />

// with {children}
<SectionHeader>Content here</SectionHeader>
```